airbyte-integrations/connectors/source-linnworks/source_linnworks/streams.py

```diff
@@ -237,3 +241,47 @@ def paged_result(self, response: requests.Response) -> Mapping[str, Any]:
     def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
         for record in self.paged_result(response)["Data"]:
             yield record
+
+    def request_cache(self) -> Cassette:
+        try:
+            os.remove(self.cache_filename)
+        except FileNotFoundError:
+            pass
+
+        return vcr.use_cassette(
+            self.cache_filename,
+            record_mode="new_episodes",
+            serializer="yaml",
+            match_on=["method", "scheme", "host", "port", "path", "query", "body"],
```

Comment:
Do you confirm that you override the `HttpStream.request_cache` to `match_on` request body, which is not the default for vcr? A small comment here might help.